distributed/utils.py

```diff
@@ -412,14 +422,14 @@ def wait(timeout):
             raise
 
     loop.add_callback(f)
```

Comment:
def cancel and def wait are missing annotations

Reply:
Addressed.